packages/frontend/components/scaffold-eth/Contract/DisplayVariables.tsx

```diff
@@ -0,0 +1,49 @@
+import { ContractFunction } from "ethers";
+import { FunctionFragment } from "ethers/lib/utils";
+import React, { FC, SetStateAction, useCallback, useEffect, useState, Dispatch } from "react";
+// import { useIsMounted } from "usehooks-ts";
+import { tryToDisplay } from "./displayUtils";
+
+interface IDisplayVariableProps {
+  contractFunction: ContractFunction | undefined;
+  functionInfo: FunctionFragment;
+  refreshRequired: boolean;
+  setTriggerRefresh: Dispatch<SetStateAction<boolean>>;
+}
+
+// TODO Check for the need of useIsMounted
+
+export const DisplayVariable: FC<IDisplayVariableProps> = props => {
```

Comment:
I think we should avoid the use of "props", even if we have `IDisplayVariableProps`.

Let's have it listed {contractFunction, functionInfo...}